app/assets/templates/gauge_template.html.erb

```diff
@@ -0,0 +1,107 @@
+<span ng-controller="GaugeCtrl">
+  <div class="widget_wrapper panel panel-default pull-left">
+    <error-message error-messages="errorMessages"></error-message>
+    <div class="js_widget_wrapper widget_title panel-heading" ng-mouseenter="showControls = true" ng-mouseleave="showControls = false">
+      <span ng-show="dashboard"><a href="/{{dashboard.slug}}" class="dashboard_link pointer">{{dashboard.name}}</a> &mdash; </span>
+      {{title()}}
+
+      <div class="btn-group pull-right graph_config_menu">
+        <div ng-show="showControls || showTab || requestsInFlight" class="btn btn-primary btn-sm" ng-click="refreshGraph()" tooltip-append-to-body="true" tooltip="Refresh"><i ng-class="{spin:requestInFlight}" class="glyphicon glyphicon-refresh"></i></div>
+        <div ng-show="showControls || showTab" class="btn btn-primary btn-sm" ng-click="toggleTab($event, 'datasources')" tooltip-append-to-body="true" tooltip="Datasources"><i class="icon-add-to-list"></i></div>
+        <div ng-show="showControls || showTab" class="btn btn-primary btn-sm" ng-click="toggleTab($event, 'graph')" tooltip-append-to-body="true" tooltip="Chart settings"><i class="icon-pie-chart"></i></div>
+        <div ng-show="showControls || showTab" class="btn btn-primary btn-sm" ng-click="toggleTab($event, 'palette')" tooltip-append-to-body="true" tooltip="Palette"><i class="icon-palette"></i></div>
```

Comment:
I think this won't need a palette tab at all, right? Right now it just says that it's not implemented yet.